docker-storage-setup.sh

```diff
@@ -1112,6 +1112,15 @@ fi
 if [ -e /etc/sysconfig/docker-storage-setup ]; then
   source /etc/sysconfig/docker-storage-setup
 fi
+if [ -z "$ROOT_VOLUME" ]; then
+    ROOT_VOLUME="$DOCKER_ROOT_VOLUME"
+fi
```

Comment:
Since you have already sourced `/usr/lib/docker-storage-setup/docker-storage-setup`: `ROOT_VOLUME=no` (default value) has been set. So If I as a user set `DOCKER_ROOT_VOLUME=yes` in `/etc/sysconfig/docker-storage-setup`, `ROOT_VOLUME` will still be `no` and this condition will never get executed.